src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/ReportHelper.java

```diff
@@ -53,35 +57,31 @@ public static boolean validateMatchingDate(ListingData listingData, String match
         }
     }
 
-    public static ListingData processBroughtForwardDatesRequest(ListingDetails listingDetails,
-                                                                List<SubmitEvent> submitEvents) {
-        if (submitEvents != null && !submitEvents.isEmpty()) {
-            log.info(CASES_SEARCHED + submitEvents.size());
-            List<BFDateTypeItem> bfDateTypeItems = new ArrayList<>();
-            for (SubmitEvent submitEvent : submitEvents) {
-                addBfDateTypeItems(submitEvent, listingDetails, bfDateTypeItems);
-            }
-            listingDetails.getCaseData().setBfDateCollection(bfDateTypeItems);
+    public static String getDurationText(ListingData currentCaseData) {
+        if (Strings.isNullOrEmpty(currentCaseData.getHearingDateType())) {
+            return null;
         }
 
-        listingDetails.getCaseData().clearReportFields();
-        return listingDetails.getCaseData();
+        if (currentCaseData.getHearingDateType().equals(SINGLE_HEARING_DATE_TYPE)) {
+            return "On " + currentCaseData.getListingDate();
+        } else {
+            return "Between " + currentCaseData.getListingDateFrom()
+                + " and " + currentCaseData.getListingDateTo();
+        }
     }
 
-    private static void addBfDateTypeItems(
-            SubmitEvent submitEvent,
-            ListingDetails listingDetails,
-            List<BFDateTypeItem> bfDateTypeItems) {
-        if (submitEvent.getCaseData().getBfActions() != null
-                && !submitEvent.getCaseData().getBfActions().isEmpty()) {
-            for (BFActionTypeItem bfActionTypeItem : submitEvent.getCaseData().getBfActions()) {
-                var bfDateTypeItem = getBFDateTypeItem(bfActionTypeItem,
-                        listingDetails.getCaseData(), submitEvent.getCaseData());
-                if (bfDateTypeItem.getValue() != null) {
-                    bfDateTypeItems.add(bfDateTypeItem);
-                }
-            }
+    public static String getFormattedLocalDate(String date) {
```

Comment:
This method really needs a unit test to document the inputs and outputs that it is expected to handle. Ideally you would use a parameterized test

Reply:
Unit tests added.